components/Account/index.js

```diff
@@ -30,87 +35,136 @@ import PaymentSources from './PaymentSources'
 
 import { APP_OPTIONS } from '../../lib/constants'
 
+import { HEADER_HEIGHT_MOBILE, HEADER_HEIGHT } from '../constants'
+
 const { H2, P } = Interaction
 
-const Account = ({ loading, error, me, t, query, hasMemberships, acceptedStatue, recurringAmount, hasPledges, merci, inNativeIOSApp }) => (
-  <Loader
-    loading={loading}
-    error={error}
-    render={() => {
-      if (!me) {
-        return (
-          <MainContainer>
-            <Content>
-              <H1>{t('account/signedOut/title')}</H1>
-              <P>
-                {t('account/signedOut/signIn')}
-              </P>
-              <SignIn email={query.email} />
-            </Content>
-          </MainContainer>
-        )
+const styles = {
+  accountAnchor: css({
+    display: 'block',
+    visibility: 'hidden',
+    position: 'relative',
+    top: -(HEADER_HEIGHT_MOBILE + 20),
+    [mediaQueries.mUp]: {
+      top: -(HEADER_HEIGHT + 20)
+    }
+  })
+}
+
+const AccountAnchor = ({ children, id }) => {
+  return (
+    <div style={{marginBottom: 80}}>
+      <a {...styles.accountAnchor} id={id} />
+      {children}
+    </div>
+  )
+}
+
+class Account extends Component {
+  componentDidMount () {
+    const match = /#(\w+)$/m.exec(this.props.router.asPath)
```

Comment:
`location.hash` should do:
`window.location.hash.substr(1)` -> `''` or `'teilen'`